jellyfin-model/src/commonMain/kotlin-generated/org/jellyfin/sdk/model/api/MediaStreamType.kt

```diff
@@ -24,8 +24,6 @@ public enum class MediaStreamType(
 	SUBTITLE("Subtitle"),
 	@SerialName("EmbeddedImage")
 	EMBEDDED_IMAGE("EmbeddedImage"),
-	@SerialName("Data")
-	DATA("Data"),
```

Comment:
Looks like we forgot to revert the revert PR (https://github.com/jellyfin/jellyfin/pull/8480)... this change will break compat with 10.8.5